_posts/2021-03-05-OpenSSL-followup.markdown

```diff
@@ -0,0 +1,20 @@
+---
+title: OpenSSL switch in progress
+layout: post
+---
+
+The previously noted [OpenSSL switch](https://voidlinux.org/news/2021/02/OpenSSL.html) 
+is now underway. Because OpenSSL is a critical component of a large number of
+packages, the rebuild process is expected to take several days. Syncing between
```

Comment:
This mixes the causality a bit, I think. it being critical or not doesn't influence the rebuild taking a long time, because it being a dependency of a bunch of packages that's the issue.

Reply:
Reworded to drop critical